tests/rogue_tests.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+PKG = 'herbpy'
+import roslib; roslib.load_manifest(PKG)
+import numpy, unittest
+import herbpy, prpy
+
+env, robot = herbpy.initialize(sim=True)
+robot.right_arm.SetActive()
+fuze = prpy.rave.add_object(env, 'fuze_bottle', 'objects/fuze_bottle.kinbody.xml')
+
+class RogueTest(unittest.TestCase):
+    def setUp(self):
+        self.env, self.robot, self.fuze = env, robot, fuze
+        self.fuze_pose = numpy.eye(4)
+        self.fuze_pose[0:3, 3] = [0.8, -0.3, 0.4]
+        self.fuze.SetTransform(self.fuze_pose)
+        self.fuze.Enable(True)
```

Comment:
Once you move the initialization here, you'll need to call `env.Destroy()` in `tearDown`.